src/pages/settings/Wallet/ReportVirtualCardFraudPage.tsx

```diff
@@ -68,7 +71,7 @@ function ReportVirtualCardFraudPage({
                 title={translate('reportFraudPage.title')}
                 onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WALLET_DOMAINCARD.getRoute(cardID))}
             />
-            <View style={[styles.flex1, styles.justifyContentBetween]}>
+            <View style={[styles.flex1, styles.justifyContentBetween, !paddingBottom ? styles.pb5 : null]}>
```

Comment:
I feel like the gap in the bottom feels too much. It didn't happen before, so might be because this [PR](https://github.com/Expensify/App/pull/38998) merged last week.

![Screenshot 2024-05-09 at 02 05 44](https://github.com/Expensify/App/assets/25520267/5956f0ef-8c45-448b-88d0-297854d70c8a)

Reply:
Yes, sorry, I've missed this one 😓 

Pushing a fix for this